scene_select/check_ancillary.py

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env python3
+
+import datetime
+from pathlib import Path
+import structlog
+import h5py
+import numpy
+import pandas
+import pytz
+
+
+LOG = structlog.get_logger()
+DEFINITIVE_START_DATE = datetime.datetime(2002, 7, 1)
+BRDF_DIR = "/g/data/v10/eoancillarydata-2/BRDF/MCD43A1.006"
+WV_DIR = "/g/data/v10/eoancillarydata-2/water_vapour"
+WV_FMT = "pr_wtr.eatm.{year}.h5"
+
+
+def read_h5_table(fid, dataset_name):
```

Comment:
This func looks familiar :smile:

Reply:
Ha ha. I pulled it out of wagl since I didn't want the dependency.